Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift

```diff
@@ -66,8 +66,11 @@ extension Sequence<InlineMarkup> {
             return nil
         }
         let remainingInitialText = initialText.suffix(from: initialText.index(after: colonIndex)).drop { $0 == " " }
+        
+        let remainingCapitalizedInitialText = remainingInitialText.capitalizeFirstWord()
+//        let remainingCapitalizedInitialText = remainingInitialText
```

Comment:
Clean this up before merge. Yes, you can probably just call it below, and even eliminate the call to the String initializer below because `capitalizeFirstWord` returns a new string already.